src/order/galois_connection.lean

```diff
@@ -525,6 +548,10 @@ lemma u_infi_l [complete_lattice α] [complete_lattice β] (gi : galois_coinsert
   u (⨅ i, l (f i)) = ⨅ i, (f i) :=
 gi.dual.l_supr_u _
 
+lemma u_Inf_l [complete_lattice α] [complete_lattice β] (gi : galois_coinsertion l u) (s : set α) :
```

Comment:
This feels like it should have image in the name

Reply:
Agreed, thanks!